README.md

```diff
@@ -206,6 +213,9 @@ The shared storage worklet invocation methods (`addModule`, `createWorklet`, and
             *   The approximate size of the contents of this interest group, in bytes.
     *   The [AuctionAdInterestGroup](https://wicg.github.io/turtledove/#dictdef-auctionadinterestgroup)'s [lifetimeMs](https://wicg.github.io/turtledove/#dom-auctionadinterestgroup-lifetimems) field will remain unset. It's no longer applicable at query time and is replaced with attributes `timeSinceGroupJoinedMs` and `lifetimeRemainingMs`.
     *   This API provides the Protected Audience buyer with a better picture of what's happening with their users, allowing for Private Aggregation reports.
+*   `navigator.locks.request(resource, callback)` and `navigator.locks.request(resource, options, callback)`
```

Comment:
Does `navigator.locks` also return a LockManager as it does in a document?

Reply:
Yes. Clarified in the text.